pkg/graph/parser/cel.go

```diff
@@ -78,6 +101,7 @@ func extractExpressions(str string) ([]string, error) {
 		expressions = append(expressions, expr)
 		start = endIdx + 1
 	}
+	fmt.Println("Expressions: ", expressions)
```

Comment:
@dirien  Looks like this debug statement was left in :)

Reply:
@a-hilaly oops!